docs/importing_astropy.rst

```diff
@@ -1,16 +1,16 @@
-*********************************
-Importing astropy and subpackages
-*********************************
+**********************************
+Importing `astropy` and sub-packages
+**********************************
```

Comment:
Here, the "***" have to extend as far as the text (the circleCI test failure is because of this)